install.sh

```diff
@@ -59,13 +61,42 @@ function install_apt {
     fi
 }
 
+function install_yum {
+    local CMD=""
+    if [ -n "${YUM_DNF}" ]; then
+        CMD=$YUM_DNF
+    else
+        if [ -n "${YUM_YUM}" ]; then
+            CMD=$YUM_YUM
+	fi
+    fi
+
+    if [ "${CMD}" != "" ]; then
+        local PARAMS="--assumeyes"
```

Comment:
Oh sorry I misread - that's in `install_yum`. I thought it was passing the same params to both `yum` and `apt-get`. Looks fine.